postgres-driver/application.go

```diff
@@ -11,18 +11,20 @@ import (
 
 const (
 	selectApplications = `
```

Comment:
I think we need to add the status field here too

Reply:
Ok will add.